src/components/organisms/Navbar/Navbar/Navbar.tsx

```diff
@@ -1,84 +1,199 @@
-import React from 'react';
+import React, { useState } from 'react';
 import styled from 'styled-components';
+import Headroom from 'react-headroom';
+import { faBars } from '@fortawesome/free-solid-svg-icons';
 
 import { colors } from '../../../../constants/colors';
-import { navbarHeight } from '../../../../constants/components';
-import useScrollThreshold from '../useScrollThreshold/useScrollThreshold';
-import FlexContainer from '../../../layout/FlexContainer/FlexContainer';
+import { breakpoints } from '../../../../constants/breakpoints';
+import { spacing } from '../../../../constants/spacing';
+import { navbarHeight, mobileNavbarHeight } from '../../../../constants/components';
+import { useViewport } from '../../../../hooks/useViewport';
+import navCurve from '../../../../content/images/nav-curve.svg';
+import Logo from '../../../atoms/Logo/Logo';
+import Icon from '../../../atoms/Icon/Icon';
+import Navbar from '../';
+export interface INavbarProps {
+  /**
+   * allows you to overlay the logo with a button or link
+   */
+  overlayLogoWith?: React.ReactNode;
+  /**
+   * Display CTA
+   */
+  withCTA?: boolean;
+  /**
+   * CTA component
+   */
+  cta?: React.ReactNode;
+}
+
+export type ILargeDeviceNavbar = Pick<INavbarProps, 'overlayLogoWith'>;
 
-export interface ILayoutOuterProps extends React.HTMLAttributes<HTMLDivElement> {
-  backgroundColor: string;
-  overlap: boolean;
+export interface IHamburgerContainerProps extends React.HTMLAttributes<HTMLSpanElement> {
+  open: boolean;
 }
 
-const LayoutOuter = styled.div<ILayoutOuterProps>`
-  position: fixed;
-  top: 0;
-  left: 0;
-  right: 0;
-  z-index: 1;
-  background-color: ${({ backgroundColor }) => backgroundColor};
-  height: ${navbarHeight}px;
-  transition: box-shadow 0.2s ease-in-out;
-  box-shadow: 0 0 0 transparent;
-  ${({ overlap }) => overlap && `box-shadow: rgba(0, 0, 0, 0.2) 0 1px 2px;`}
+const PageNavigation = styled.header`
+  .headroom {
+    top: 0;
+    left: 0;
+    right: 0;
+    z-index: 1;
+    background-color: ${colors.brand};
+    height: ${mobileNavbarHeight}px;
+
+    @media (min-width: ${breakpoints.desktop}px) {
```

Comment:
Instead of this I think you can use the `minMedia` helper from `helpers/responsiveness.ts`:
```
  ${minMedia.desktop`
  /* your css */
`
```